### DISCUSSION TITLE
Budget Setup

### DISCUSSION THREAD

**Discussion by vomikan** [moneymanagerex/moneymanagerex#6192] [General]
Hi there! I hope some of you can help me with this. I'm trying to setup the budget and I found that the selected month is not matching with the actual month. For example in the screenshot you can see that I select month 2 and this is sync with transaction from August as you can see between the brakets. I'm doing something wrong or is a bug? Thanks.

![image](https://github.com/moneymanagerex/moneymanagerex/assets/6836805/7fa590c7-e818-4cfb-bd6c-c7e9556fc349)

https://moneymanagerex.slack.com/archives/C1FL7FJR4/p1692869889982749

**Comment by Dorusto** [comment]
Apparently this portable mmex found on this [post](https://github.com/moneymanagerex/moneymanagerex/issues/6040#issuecomment-1603976222) fix my problem also! Thanks!
